### PR TITLE
extend support for python3 identifiers

### PR DESCRIPTION
**Pull request by s-cork**
closes #1237, #1176

### CODE REVIEW

**Comment by darth83** [comment]
Any estimate on when this will be merged? Kind of need it soon...

File: support/polyfills/Unicode.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
...\\u12c8-\\u12d6\\u12d8-\\u1310\\u1312-\\u1315\\u1318-\\u135a\\u1380-\\u138f\\u13a0-\\u13f4\\u1401-\\u166c\\u166f-\\u1676\\u1681-\\u169a\\u16a0-\\u16ea\\u1700-\\u170c\\u170e-\\u1711\\u1720-\\u1731\\u1740-\\u1751\\u1760-\\u176c\\u176e-\\u1770\\u1780-\\u17b3\\u17dc\\u1820-\\u1842\\u1844-\\u1877\\u1880-\\u18a8\\u1900-\\u191c\\u1950-\\u196d\\u1970-\\u1974\\u1980-\\u19a9\\u19c1-\\u19c7\\u1a00-\\u1a16\\u1b05-\\u1b33\\u1b45-\\u1b4b\\u2135-\\u2138\\u2d30-\\u2d65\\u2d80-\\u2d96\\u2da0-\\u2da6\\u2da8-\\u2dae\\u2db0-\\u2db6\\u2db8-\\u2dbe\\u2dc0-\\u2dc6\\u2dc8-\\u2dce\\u2dd0-\\u2dd6\\u2dd8-\\u2dde\\u3006\\u303c\\u3041-\\u3096\\u309f\\u30a1-\\u30fa\\u30ff\\u3105-\\u312c\\u3131-\\u318e\\u31a0-\\u31b7\\u31f0-\\u31ff\\u3400\\u4db5\\u4e00\\u9fbb\\ua000-\\ua014\\ua016-\\ua48c\\ua800\\ua801\\ua803-\\ua805\\ua807-\\ua80a\\ua80c-\\ua822\\ua840-\\ua873\\uac00\\ud7a3\\uf900-\\ufa2d\\ufa30-\\ufa6a\\ufa70-\\ufad9\\ufb1d\\ufb1f-\\ufb28\\ufb2a-\\ufb36\\ufb38-\\ufb3c\\ufb3e\\ufb40\\ufb41\\ufb43\\ufb44\\ufb46-\\ufbb1\\ufbd3-\\ufd3d\\ufd50-\\ufd8f\\ufd92-\\ufdc7\\ufdf0-\\ufdfb\\ufe70-\\ufe74\\ufe76-\\ufefc\\uff66-\\uff6f\\uff71-\\uff9d\\uffa0-\\uffbe\\uffc2-\\uffc7\\uffca-\\uffcf\\uffd2-\\uffd7\\uffda-\\uffdc",
```
Comment (furka) [inline support/polyfills/Unicode.js:114]: I believe there is a `-` missing between `u4e00` and `\\u9fbb`, it probably should be `u4e00-\\u9fbb`
Without this `-` it no longer matches full range chinese/japanese/korean ideographs, but only the first and last one.

`第一行 = 5` should work, but doesn't due to the missing `-`

Comment (s-cork) [reply support/polyfills/Unicode.js]: I've updated the values using https://github.com/slevithan/xregexp/blob/master/tools/output/categories.js
thanks for spotting

**Review by meredydd — APPROVED** [review]
Looks good to me! Let's get this merged.

**Comment by meredydd** [comment]
Hi @skulpt/committers - this looks like a fairly high priority change. I think it's probably small enough to merge myself, but I'll give people a couple of days to look at it in case you want to get a word in :)

File: src/tokenize.js [hunk 2]
```diff
@@ -79,32 +81,17 @@ function rstrip (input, what) {
...
-        return new RegExp('^' + id_start + '+' + id_continue + '*$', 'u');
-    } else {
-        id_start = group(Lu, Ll, the_underscore);
-        id_continue = group(id_start, '[0-9]');
-        return new RegExp('^' + id_start + '+' + id_continue + '*$');
-    }
-})();
+
+
+const { Lu, Ll, Lt, Lm, Lo, Nl, Mn, Mc, Nd, Pc } = Unicode;
```
Comment (albertjan) [inline src/tokenize.js:86]: 🤯 🤦 👍

**Review by albertjan — APPROVED** [review]

**Pull request merged**